packages/utils/src/date/index.ts

```diff
@@ -81,7 +81,7 @@ export function formatReadableUTCOffset(offset: number): string {
  * @see https://github.com/prantlf/date-fns-timezone/blob/master/src/formatToTimeZone.js#L131
  */
 function formatTimeZoneTokens(dateFormat: string, timeZone: string): string {
-	return dateFormat.replace(/(?<!\[)(z|ZZ?)/g, match => {
+	return dateFormat.replace(/(z|ZZ?)(?!\])/g, match => {
```

Comment:
There's already a unit test for this logic:
https://github.com/Talend/ui/blob/master/packages/utils/src/date/date.test.ts#L119